beacon-chain/monitor/process_attestation.go

```diff
@@ -0,0 +1,184 @@
+package monitor
+
+import (
+	"fmt"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/config/params"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/attestation"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/block"
+	"github.com/prysmaticlabs/prysm/runtime/version"
+	"github.com/prysmaticlabs/prysm/time/slots"
+	"github.com/sirupsen/logrus"
+)
+
+// getAttestingIndices returns the indices of validators that appear in the
+// given aggregated atestation.
+func (s *Service) getAttestingIndices(state state.BeaconState, att *ethpb.Attestation) ([]uint64, error) {
```

Comment:
```suggestion
func attestingIndices(state state.BeaconState, att *ethpb.Attestation) ([]uint64, error) {
```

dont need a receiver for this, and for idiomatic Go we tend to not prefix with `get`

Reply:
I need a context to pass the Committee getter

Reply:
Removed the service